### ISSUE TITLE
Fail to serve because of final classes in react-php

### ISSUE THREAD

**Issue by gabrieliuga** [beyondcode/laravel-websockets#222]
The new version of react/dns has made all classes final which breaks websockets.
https://github.com/reactphp/dns/commit/eae321547557703d8c92bbf272295b8681a26e88#comments

```
PHP Fatal error:  Class BeyondCode\LaravelWebSockets\Statistics\DnsResolver 
may not inherit from final class (React\Dns\Resolver\Resolver) in 
vendor/beyondcode/laravel-websockets/src/Statistics/DnsResolver.php on line 8

   Symfony\Component\Debug\Exception\FatalErrorException  : Class 
BeyondCode\LaravelWebSockets\Statistics\DnsResolver 
may not inherit from final class (React\Dns\Resolver\Resolver)
```
The dependency on react/* packages comes from clue/buzz-react

**Comment by deviouspk** [comment]
If you want an easy fix until the package developers merge this pull request #223. You can add this to your composer file:
"react/dns" : "0.4.19"

**Comment by francislavoie** [comment]
This can be closed now that the PRs have been merged.

**Comment by gabrieliuga** [comment]
closed in #223
